### ISSUE TITLE
Request: Cartoon Network

### ISSUE THREAD

**Issue by ILoveHDDark** [simple-icons/simple-icons#8450]
### Brand Name

Cartoon Network

### Website

https://cartoonnetwork.com

### Popularity Metric

The Similarweb rank is 112,097. See https://www.similarweb.com/website/cartoonnetwork.com

### Official Resources for Icon and Color

svg: https://upload.wikimedia.org/wikipedia/commons/8/80/Cartoon_Network_2010_logo.svg
color: `#000000`  taken from the site header

### Contributing

Yes

**Comment by uncenter** [comment]
I tried adding this a month or two ago but ran into an issue where I needed to have extra white space on one side in order for the logo to be even. The logo consists of two letters, C and N, the left letter being C and surrounded by a black square and the right letter being N and surrounded by a "white square", which is really just empty space... But then the linter complains about the SVG not being sized correctly if I leave the correct space to the right of the N:

![Screenshot 2024-06-04 at 17 39 58 (Inkscape)](https://github.com/simple-icons/simple-icons/assets/47499684/5f86dca0-7760-4fbd-bdd2-d68d005e2df8)

**Comment by anhthang** [comment]
Including text under the logo seems necessary.

**Comment by uncenter** [comment]
Seems necessary in general or to preserve the proportions as per my previous comment? The text is so small it won't be readable in the slightest at a badge size.

**Comment by anhthang** [comment]
I agree the text is a bit small, but for now it's necessary to maintain vertical centering between the C and N squares. Additionally, the text itself isn't centered vertically either.

**Comment by anhthang** [comment]
@uncenter just saw `.svglint-ignored.json` file in the repo, might be helpful in this case

**Comment by adamrusted** [comment]
> @uncenter just saw `.svglint-ignored.json` file in the repo, might be helpful in this case

Unfortunately not, the Lint ignore file is just for when an icon _does_ match our criteria but the linter still has issues.
In this case, I'd be inclined to say we can't include the icon if we can't match it to our criteria.

**Comment by anhthang** [comment]
The brand seems recognizable, but the text and logo aren't vertically centered. This creates an unbalanced look, in my opinion.

![Cartoon Network](https://github.com/simple-icons/simple-icons/assets/10458061/1f6d5e54-fe8e-4fd2-a5ea-0dfebc336037)
